core/src/fees.ts

```diff
@@ -15,16 +21,21 @@ export const getApproximateTransactionFees = async function (network: string): P
 
     // TODO: figure out a way to properly estimate gas
     // for each transaction when no wallet is connected
-    const bidTransactionFeeETH = gasPrice.multipliedBy(145438);
-    const swapTransactionFeeETH = gasPrice.multipliedBy(722651);
-    const authTransactionFeeETH = gasPrice.multipliedBy(48356);
-    const restartTransactionFeeETH = gasPrice.multipliedBy(209182);
+    const bidTransactionFeeETH = gasPrice.multipliedBy(BID_TRANSACTION_GAS_LIMIT);
+    const swapTransactionFeeETH = gasPrice.multipliedBy(SWAP_TRANSACTION_GAS_LIMIT);
+    const swapTransactionVariableFeeETH = gasPrice.multipliedBy(
+        SWAP_TRANSACTION_GAS_LIMIT - BID_TRANSACTION_GAS_LIMIT
+    );
+    const authTransactionFeeETH = gasPrice.multipliedBy(AUTH_TRANSACTION_GAS_LIMIT);
+    const restartTransactionFeeETH = gasPrice.multipliedBy(RESTART_TRANSACTION_GAS_LIMIT);
 
     return {
         bidTransactionFeeETH,
         bidTransactionFeeDAI: await convertETHtoDAI(network, bidTransactionFeeETH),
         swapTransactionFeeETH,
         swapTransactionFeeDAI: await convertETHtoDAI(network, swapTransactionFeeETH),
+        swapTransactionVariableFeeETH,
+        swapTransactionVariableFeeDAI: await convertETHtoDAI(network, swapTransactionVariableFeeETH),
```

Comment:
I would also not include this new key into the auction object, since it's just for the internal purpose. Once again: the transaction fee consist of 

- A. exchanging one currency into another
- B. executing the auction

`B.` is approximately equals to `bidTransactionFeeETH`. Previously we included `A.` into `B.` and always displayed them together, but now we want to split those two and have a `A.` specified per callee/market. This means, you can add those keys to the `MarketDataBase` and create a small function, eg `getDefaultMarketFee` that is called in the `enrichAuctionWithMarketData` and  just returns those fees under `exchangeFeeEth` and `exchangeFeeDai`